MediaPlayer/MediaControls.module.less

```diff
@@ -8,64 +8,99 @@
 	text-align: center;
 	direction: ltr;
 	white-space: nowrap;
-	bottom: 42px;
 
 	> *:first-child {
 		margin-inline-start: 0;
 	}
 }
 
 .applySkins({
-	.button {
-		height: @agate-mediacontrols-button-height;
-		min-width: @agate-mediacontrols-button-min-width;
-		margin: @agate-mediacontrols-button-margin;
-
-		.bg {
-			border: @agate-mediacontrols-button-bg-border;
-			height: @agate-mediacontrols-button-bg-height;
-			width:  @agate-mediacontrols-button-bg-width;
-		}
-
-		&,
-		.client {
-			width: @agate-mediacontrols-button-client-width;
-		}
+	.controlsFrame {
+		bottom: @agate-mediacontrols-bottom;
 
-		.icon {
-			padding: @agate-mediacontrols-button-icon-padding;
-		}
-
-		&.playPauseButton {
-			height: @agate-mediacontrols-playpause-button-height;
-			padding: @agate-mediacontrols-playpause-button-padding;
+		.button {
+			height: @agate-mediacontrols-button-height;
+			min-width: @agate-mediacontrols-button-min-width;
+			.margin-start-end(@agate-mediacontrols-button-margin);
+			width: @agate-mediacontrols-button-bg-width;
```

Comment:
Do we need this variable or can we use `@agate-mediacontrols-button-size` for all?
Honestly, I'm very uncomfortable with this variation... I thought GUI will give the same size as she defined in the Button component. But implementing like this, it's very hard to maintain, and also it loses unity.

Reply:
But I'm not blaming you... we should've been communicated with GUI better.

Reply:
You are right. I shall unify height and width variables